src/main/java/com/google/sps/data/InterviewPostRequest.java

```diff
@@ -18,18 +18,30 @@
 
 /**
  * An InterviewPostRequest is used to schedule an interview for the current user with an interviewer
- * who has the selected company and job and is available at the specified time, which is represented
- * by the utcStartTime String.
+ * who is qualified to interview for the selected position, has the selected company and job, and is
+ * available at the specified time, which is represented by the utcStartTime String.
  */
 public class InterviewPostRequest {
   private String company;
   private String job;
   private String utcStartTime;
+  private String position;
 
-  public InterviewPostRequest(String company, String job, String utcStartTime) {
+  public InterviewPostRequest(String company, String job, String utcStartTime, String position) {
     this.company = company;
     this.job = job;
     this.utcStartTime = utcStartTime;
+    this.position = position;
+  }
+
+  public void fixNotSpecifiedFields() {
+    if (company.equals("<Not specified>")) {
+      company = "";
+    }
+    if (job.equals("<Not specified>")) {
+      job = "";
+    }
+    return;
   }
```

Comment:
Personally l feel like this should all be done in the Javascript. Backend should deal with "clean" data
@acmcarther @jalexanderqed

Reply:
Jack asked for this earlier, I'll take the time to do that now.

Reply:
I'm not sure how to do this with the modal. Will ask at sync.

Reply:
Done